src/System.Drawing.Common/src/System/Drawing/Region.Windows.cs

```diff
@@ -17,8 +17,8 @@ public void ReleaseHrgn(IntPtr regionHandle)
                 throw new ArgumentNullException(nameof(regionHandle));
             }
 
-            // On Windows HRGN are (old) GDI objects. Deskop .NET does not check the return code of IntDeleteObject
-            SafeNativeMethods.IntDeleteObject(new HandleRef(this, regionHandle));
+            // On Windows HRGN are (old) GDI objects. Deskop .NET does not check the return code of DeleteObject
```

Comment:
I don't think this comment is needed, but if we want to keep it we should change `Deskop .Net` to `.NET Framework`.